components/appointmentPicker/appointmentPickerController.js

```diff
@@ -97,7 +106,7 @@ define('appointmentPickerController', [], function() {
 				return (time + ' - FULL' + ($scope.appointmentPickerSharedProperties.isLoggedIn ? ' - overscheduled by ' + Math.abs(info.appointmentsAvailable) + ' appointments' : ''));
 			} else { // Virtual site
 				if (appointmentsStillAvailable) {
-					return 'No particular time--Your appointment will occur sometime during the week of the selected date.'
+					return time;
 				}
 				return ('No appointments available during the week of the selected date - FULL' + ($scope.appointmentPickerSharedProperties.isLoggedIn ? ' - overscheduled by ' + Math.abs(info.appointmentsAvailable) + ' appointments' : ''))
```

Comment:
Is it worth changing the text of this too?

Reply:
This just shows time now--I made this change on the server and didn't delete the wording here.